mvp-spec/publishers-requirements.md

```diff
@@ -151,10 +123,10 @@ Here is a JSON example of the Seed:
 ```
 <!--partial-end-->
 
-The Ad Server must sign the Seeds ("source"."signature"). The Elliptic Curve Digital Signature Algorithm
+The Publisher must sign the Seeds ("source"."signature"). The Elliptic Curve Digital Signature Algorithm
```

Comment:
Reference to ECDA should be removed to restrain mentions to it to signatures.md file

Reply:
Agreed, updated.